frontend/src/app/design-switcher/components/design-switcher/design-switcher.component.scss

```diff
@@ -0,0 +1,57 @@
+// Based on this example
+
+.container {
+    display: block;
+    position: relative;
+    padding-left: 35px;
+    margin-bottom: 12px;
+    cursor: pointer;
+    font-size: 22px;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+  
+  .container input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+  }
+  
+  .checkmark {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 25px;
+    width: 25px;
+    background-color: white;
```

Comment:
Can we change the default color to something with a little more contrast to the background or add a small border to the radio button? Depending on the display's contrast, the clickable area isn't visible very well.